gpus/getting-started-gpus.html.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Getting Started with Fly GPUs
+layout: docs
+nav: firecracker
+---
+
+## How do I run a GPU Machine on Fly.io?
+
+A Fly GPU Machine works very similarly to a CPU-only (or "normal") Fly Machine, and has access to the same platform features by default. It boots up with GPU drivers installed and you can run `nvidia-smi` right away. So to run a Fly App that uses CUDA compute is a slight variation on running any Fly App. In a nutshell:
+
+1. Make sure GPUs are enabled on your Fly Organization.
+1. Tell the Fly Platform to provision your Machines on Fly GPU hosts.
+1. Provide a Docker image or Dockerfile for your project that installs the NVIDIA libraries your project uses, alongside the rest of your project.
+1. Carry on as you would for any Fly App, tailoring it to the needs of your application.
+
+<div class="note icon">
+**Aside:** Firecracker doesn't do GPUs, so GPU-enabled Machines run under Cloud Hypervisor instead. This means you'll see some different output in logs, but in general you don't have to think about this.
+</div>
+
+## Placing your Machines on GPU hosts
+
+Not all of Fly.io's physical servers have GPUs attached to them, so you have to tell the platform your requirements before the Machine is created on a host.
+
+Configuration is done by different means depending on whether you're using Fly Launch to manage your app, or running Machines individually using the API or the [`fly machine run` command](/docs/machines/run/).
```

Comment:
So the config is the same whether you use flags with `fly m run`, send a config in the body of an API request, or put it in `fly.toml` and run `fly deploy`.  I want to avoid making that ambiguous.

Reply:
Keeping this for refinement later